python/graphstorm/model/lm_model/hf_bert.py

```diff
@@ -16,43 +16,32 @@
     Huggingface bert support
 """
 import time
-import inspect
 
 import numpy as np
 import torch as th
 from torch import nn
 
-from transformers import BertModel, BertConfig
+from transformers import AutoModel, AutoConfig, PreTrainedModel
 
 from .lm_model import TOKEN_IDX, ATT_MASK_IDX, VALID_LEN, TOKEN_TID_IDX
 from .lm_model import GSFLanguageModelWrapper
 
-def load_hfbert_model(bert_configs):
-    """ Load huggingface bert model
+def load_hfbert_model(bert_configs: dict):
+    """ Load huggingface LM model
     """
     model_name = bert_configs["model_name"]
-    gradient_checkpointing = bert_configs["gradient_checkpoint"]
-    signature = inspect.signature(BertConfig.__init__)
```

Comment:
what is this used for? @classicsong

Reply:
This is for extracting default values of some parameters of BertConfig.__init__.